test/token/ERC20/ERC20.behavior.js

```diff
@@ -197,6 +157,50 @@ function shouldBehaveLikeERC20 (errorPrefix, initialSupply, initialHolder, recip
   });
 }
 
+function shouldBehaveLikeERC20Transfer (errorPrefix, from, to, balance, transfer) {
+  describe('when the recipient is not the zero address', function () {
+    describe('when the sender does not have enough balance', function () {
+      const amount = balance.addn(1);
+
+      it('reverts', async function () {
+        await shouldFail.reverting.withMessage(transfer.call(this, from, to, amount),
```

Comment:
I don't think this `.call` is necessary.

Reply:
`transfer` may access `this` (in some cases it's `this.token.transfer`).